### DISCUSSION TITLE
Works great for attachments, but how are "orphaned" notes defined?

### DISCUSSION THREAD

**Discussion by tefloon** [sandorex/nuke-orphans#6] [Q&A]
Hi! Thanks for a really great plugin! As mentioned in the other discussion, it is a true godsend when it comes to the attachments folder!

I just realized that I don't understand the other functionality, though. If anyone could help me out, I'd be really grateful.
So the thing is, whenever I try to `Trash orphaned notes` I get a list of **all** my notes (at least as far as I can tell).

I thought that what this command does, is find files that are not linked anywhere (without any _parents_, hence **orphaned**) 
However, I double-checked, and even the notes that have multiple outgoing links, as well as backlinks, appear on the list to be nuked.

To give an example, I do have the following structure:
```
                         /Index.md (in the main folder of the vault)
                                   |
                                   | has a link to
                                   ↓ 
                      /Free Time/Games.md
                                   |
                                   | has a link to
                                   ↓
/Free-Time/Games/Baldurs Gate/Baldurs Gate.md
```
And all three of these files (`Index.md`, `Games.md` and `Baldur's Gate.md`) appear on the list to get nuked.

So this really made me think - what does the **orphaned** mean in the context of the plugin? 
The only thing that comes to my mind, other than what I'd thought originally, would be files which are not _embedded_ anywhere (like is the case with attachments). Is that so?

Thank you in advance for any and all help in the matter!

**Comment by sandorex** [comment]
The plugin only counts links to the file/note/attachment, anyways this is a bug so please open an issue and provide a vault that behaves the same so i can find it

Someone had the same issue but with attachments and it just disappeared suddenly so i couldn't fix it
